client/js/components/map/admin/MapAdmin.vue

```diff
@@ -0,0 +1,302 @@
+<license>
+  (c) 2010-present DEMOS plan GmbH.
+
+  This file is part of the package demosplan,
+  for more information see the license file.
+
+  All rights reserved
+</license>
+
+<template>
+  <div class="layout u-pb">
+    <div class="layout__item w-1/1">
+      <dp-input
```

Comment:
Do we need to display those inputs here?

Reply:
Yes. they display the coordinates. If they have to be inputs, can be discussed. That I questioned myself.

Reply:
Where do I see that in the old twig-template? cant find that

Reply:
In the twig there was no permission, because it was not needed. With this PR we now added the page for a new project that should not have this feature (map extent).

Reply:
> Yes. they display the coordinates. If they have to be inputs, can be discussed. That I questioned myself.

Sorry for being unclear: I question the need to display the coordinates here, if we show them visually in the map.

Reply:
> In the twig there was no permission, because it was not needed. With this PR we now added the page for a new project that should not have this feature (map extent).

do you know that permission? Is there already a project PR for that?

Reply:
> do you know that permission? Is there already a project PR for that?

No, I don't know. I listed it here https://demoseurope.youtrack.cloud/issue/DPLAN-11443, but we probably need to ask BE if they added the permission.